### PR TITLE
chore: Create story issue template

### PR DESCRIPTION
**Pull request by coreyarnold**
add the story template

## Description

Please provide a brief description of the changes introduced in this pull request.
What problem does it solve? What is the context of this change?

## How to Test

Please describe how you have tested these changes. Have you run the code against an example application?
What steps did you take to ensure that the changes are working correctly?

## Related Issues

Please include any related issues or pull requests in this section, using the format `Closes #<issue number>` or `Fixes #<issue number>` if applicable.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/story.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+_What can't be achieved within this story?_
+
+### Dependencies
+_Do any other teams or parts of the New Relic product need to be considered?_
+_Some common areas: UI, collector, documentation_
+
+### Additional context
+_What else should we know about this story that might not fit into the other categories?_
+
+### Estimates
```
Comment (bizob2828) [inline .github/ISSUE_TEMPLATE/story.md:27]: do we really want to put that in the ticket?

**Review by bizob2828 — DISMISSED** [review]
Do we want in the description an example with the `As a x I <want to> <so that>`. Similar to this https://www.atlassian.com/agile/project-management/user-stories

**Comment by bizob2828** [comment]
This needs rebased as it has broken tests that have since been fixed

File: .github/ISSUE_TEMPLATE/story.md (hunk 1 above)
Comment (coreyarnold) [reply .github/ISSUE_TEMPLATE/story.md]: removed the estimate section. we can revisit later if needed.

**Review by bizob2828 — APPROVED** [review]

**Pull request merged**